MantidQt/API/src/NonOrthogonal.cpp

```diff
@@ -417,10 +419,9 @@ K = M24X + M22Y + M23Z
 L = M31X + M32Y + M33Z
 
 */
-void transformLookpointToWorkspaceCoord(Mantid::coord_t *lookPoint,
-                                        const Mantid::coord_t skewMatrix[9],
-                                        const size_t &dimX, const size_t &dimY,
-                                        const size_t &dimSlice) {
+void transformLookpointToWorkspaceCoord(
+    Mantid::coord_t *lookPoint, const std::array<Mantid::coord_t, 9> skewMatrix,
```

Comment:
That's weird. The header has it defined as a reference. I'm surprised this linked...